nav2_navfn_planner/src/navfn_planner.cpp

```diff
@@ -134,13 +138,10 @@ nav_msgs::msg::Path NavfnPlanner::createPlan(
 bool
 NavfnPlanner::isPlannerOutOfDate()
 {
-  if (!planner_.get() ||
+  return !(
```

Comment:
Please don't add changes not related to the memory leaks in here, this could have really easily been overlooked

Reply:
Ah, sure! I ended up making a couple of changes to reduce code. I'll revert the unrelated changes.